### PR TITLE
release: OKD jobs should not be in release-origin

### PR DESCRIPTION
**Pull request by smarterclayton**
OKD is not exactly CI and should use the `-openshift-okd-` naming
scheme for jobs.

### CODE REVIEW

**Comment by vrutkovs** [comment]
:+1:

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml [hunk 1]
```diff
@@ -3953,6 +3953,127 @@ periodics:
...
+  cluster: api.ci
+  decorate: true
+  interval: 48h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-env: aws
+    job-release: "4.5"
+    job-test: e2e
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-okd-installer-e2e-aws-4.5
```
Comment (vrutkovs) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:3966]: Dupe of existing job

Comment (smarterclayton) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:3966]: I don't see that?

Comment (smarterclayton) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: Nm, found it.

**Pull request merged**